lib/db/mysql.js

```diff
@@ -800,6 +807,19 @@ module.exports = function (log, error) {
     )
   }
 
+  // Select : emails
```

Comment:
`Select :` => `Update :` ?

Then the other `Update :` comments have `Set :` and `Where :` lines.